pack/tsk/tskp.json

```diff
@@ -395,6 +509,181 @@
+        "traits": "Item. Science.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "09041",
+        "cost": 2,
+        "customization_options": [
+            {
+                "xp": 1,
+                "text_change": "insert",
+                "position": 3
+            },
+            {
+                "xp": 1,
+                "text_change": "insert",
+                "position": 3
+            },
+            {
+                "xp": 1,
+                "text_change": "insert",
+                "position": 3
+            },
+            {
+                "xp": 1,
+                "text_change": "insert",
+                "position": 3
+            },
+            {
+                "xp": 2,
+                "text_change": "replace",
+                "position": 4
+            },
+            {
+                "xp": 2,
+                "text_change": "append"
+            },
+            {
+                "xp": 3,
+                "text_change": "append"
+            },
+            {
+                "xp": 4,
+                "text_change": "insert",
+                "position": 4
+            }
+        ],
+        "customization_change": "- You run out of cards in your hand.\n- You are dealt damage or horror.\n- You discard a treachery or enemy from play.\n- You enter a location with 3 or more shroud.\nWhen the chosen criteria is met by any investigator at your location, you may exhaust Empirical Hypothesis to add 1 evidence to it. Other investigators at your location may trigger [fast] abilities on Empirical Hypothesis.\n[fast] Spend 2 evidence: Reduce the cost of the next card you play by 3.\n[fast] Spend 3 evidence: Discover 1 clue at your location.\nAfter you exhaust Empirical Hypothesis, you may resolve its forced effect, choosing a criteria you have not chosen this round. Then, ready it.",
+        "customization_text": "□ <b>Pessimistic Outlook.</b> Add the following criteria:\"you run out of cards in your hand\".\n□ <b>Trial and Error.</b> Add the following criteria: \"you are dealt damage or horror.\"\n□ <b>Independent Variable.</b> Add the following criteria: \"you discard a treachery or enemy from play.\"\n□ <b>Field Research.</b> Add the following criteria: \"you enter a location with 3 or more shroud.\"\n□□ <b>Peer Review.</b> The chosen criteria is met if any investigator at your location meets it, instead of only you. Other investigators at your location may trigger [fast] abilities on Empirical Hypothesis.\n□□ <b>Research Grant.</b> Empirical Hypothesis gains: \"[fast] Spend 2 evidence: Reduce the cost of the next card you play by 3.\"\n□□□ <b>Irrefutable Proof.</b> Empirical Hypothesis gains: \"[fast] Spend 3 evidence: Discover 1 clue at your location.\"\n□□□□ <b>Alternative Hypothesis.</b> After you exhaust Empirical Hypothesis, you may resolve its forced effect, choosing a criteria you have not chosen this round. Then, ready it.",
```

Comment:
```suggestion
        "customization_text": "□ <b>Pessimistic Outlook.</b> Add the following criteria:\"you run out of cards in your hand\".\n□ <b>Trial and Error.</b> Add the following criteria: \"you are dealt damage or horror.\"\n□ <b>Independent Variable.</b> Add the following criteria: \"you discard a treachery or enemy from play.\"\n□ <b>Field Research.</b> Add the following criteria: \"you enter a location with 3 or more shroud.\"\n□□ <b>Peer Review.</b> The chosen criteria is met if any investigator at your location meets it, instead of only you. Other investigators at your location may trigger [fast] abilities on Empirical Hypothesis.\n□□ <b>Research Grant.</b> Empirical Hypothesis gains: \"[fast] Spend 2 evidence: Reduce the cost of the next card you play this phase by 3.\"\n□□□ <b>Irrefutable Proof.</b> Empirical Hypothesis gains: \"[fast] Spend 3 evidence: Discover 1 clue at your location.\"\n□□□□ <b>Alternative Hypothesis.</b> After you exhaust Empirical Hypothesis, you may resolve its forced effect, choosing a criteria you have not chosen this round. Then, ready it.",
```